### DISCUSSION TITLE
Baby Stepping - Ender 3 V2

### DISCUSSION THREAD

**Discussion by pacocatech** [mriscoc/Ender3V2S1#377] [Q&A] [answered]
On the releases page it says we can use baby stepping, where do I find this option in the ender 3 v2 menu?

**Comment by mriscoc** [comment, marked as answer]
Baby steps are implemented when Z-offset is adjusted.

**Comment by pacocatech** [comment]
I imagined it would be like this, however, it only works for me when I'm not printing, when I'm printing if I try to change the height of the Z by the offset probe the Z axis does not move =/

**Reply by mriscoc** [reply]
If you click z-offset in Tune menu and turn your knob the z axis must move while you are printing.

**Reply by pacocatech** [reply]
Yes, I understand that this should be the behavior, however, it doesn't =/

**Reply by pacocatech** [reply]
is there any logging/debug I can enable to check if z is being set?

**Reply by mriscoc** [reply]
I have a Ender3V2 and live z-offset is working, what firmware version are you using?
Is the z-offset in the dashboard changing?

**Reply by pacocatech** [reply]
your version:  Ender3V2-422-BLT-20220816.bin

**Reply by mriscoc** [reply]
Live Z-offset is working with that version, verified right now.

**Reply by pacocatech** [reply]
ok, thanks

**Comment by pacocatech** [comment]
@mriscoc 
see the video, changing the values in the offset probe and the Z axis does not move.

https://we.tl/t-pop64UPVLA

**Reply by mriscoc** [reply]
Hi, it is working ok here. Is your printer homing Z? In your video I saw the display shows X and Y axes values in 0.00 but your head is in the center of the bed, that is not correct.

https://user-images.githubusercontent.com/2745567/192879737-a4b76560-cf52-4608-8dc1-298e280b3b79.mp4
